### PR TITLE
Fixed a few obstacles faced while trying to run the web example

### PR DESCRIPTION
**Pull request by us-raph**

### CODE REVIEW

File: packages/exemplar-sample/package.json [hunk 1]
```diff
@@ -28,10 +28,12 @@
   "homepage": "https://github.com/godaddy/exemplar/tree/master/packages/exemplar-sample",
   "dependencies": {
     "@exemplar/storybook-react": "^1.1.0",
-    "@exemplar/storybook-react-native": "^1.0.1",
+    "@exemplar/storybook-react-native": "^1.0.0",
     "@storybook/addon-knobs": "^5.1.9",
     "@storybook/addons": "^5.1.9",
+    "@storybook/core": "^5.1.11",
     "@storybook/react": "^5.1.9",
+    "core-js": "^3.2.1",
```
Comment (msluther) [inline packages/exemplar-sample/package.json]: This was specifically needed? That seems a bit weird.

File: packages/exemplar-sample/package.json [hunk 2]
```diff
@@ -28,10 +28,12 @@
   "homepage": "https://github.com/godaddy/exemplar/tree/master/packages/exemplar-sample",
   "dependencies": {
     "@exemplar/storybook-react": "^1.1.0",
-    "@exemplar/storybook-react-native": "^1.0.1",
+    "@exemplar/storybook-react-native": "^1.0.0",
     "@storybook/addon-knobs": "^5.1.9",
     "@storybook/addons": "^5.1.9",
+    "@storybook/core": "^5.1.11",
```
Comment (msluther) [inline packages/exemplar-sample/package.json]: Probably best to keep this version in sync with the other storybooks. I think they version everything at the same time. So either make this ^5.1.9 or the others ^5.1.11 to match this one.

File: packages/exemplar-sample/package.json (hunk 1 above)
Comment (us-raph) [reply packages/exemplar-sample/package.json]: It was. After fixing the `@exemplar/storybook-react-native` version to one that's actually available and installing the babel presets, all the builds for the web examples started failing due to the following exception:

```
ERROR in ./examples/web/yo-dawg.js
Module not found: Error: Can't resolve 'core-js/modules/es.symbol.iterator' in '/Users/rferreira/workspace/temp/exemplar/packages/exemplar-sample/examples/web'
 @ ./examples/web/yo-dawg.js 3:0-44
 @ ./examples/web sync ^\.\/([-\w]+)\.js$
 @ ./node_modules/@exemplar/storybook-react/load-examples.js
 @ multi ./node_modules/@storybook/react/node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/react/node_modules/@storybook/core/dist/server/preview/globals.js ./examples/.setup/config.js ./node_modules/@exemplar/storybook-react/load-examples.js (webpack)-hot-middleware/client.js?reload=true
```

I fixed it by installing core-js

Comment (us-raph) [reply packages/exemplar-sample/package.json]: I can still repro. Can try other suggestions for fixing it.

File: packages/exemplar-sample/package.json (hunk 2 above)
Comment (us-raph) [reply packages/exemplar-sample/package.json]: 👍

File: packages/exemplar-sample/package.json (hunk 1 above)
Comment (msluther) [reply packages/exemplar-sample/package.json]: Weird. Seems fine though

**Comment by indexzero** [comment]
Thanks for your contribution!

**Pull request merged**